big_tests/rebar.config

```diff
@@ -10,16 +10,16 @@
 {deps, [
         {lager, "3.9.2"},
         {base16, "2.0.1"},
-        {exml, "3.2.1", {pkg, hexml}},
-        {bbmustache, "1.10.0"},
+        {exml, "3.2.2", {pkg, hexml}},
```

Comment:
There's a 3.3.3 one published 😬

Reply:
Done 😃 Thx for noticing. When I first looked at the newest versions it wasn't released yet.